w3d_translator/w3d_translator.py

```diff
@@ -44,42 +44,74 @@ def farewell():
     exit(0)
 
 
-# Translate a single project
+# Copy all files from source to destination
+def copy_files(source: Path, destination: Path):
+    # TODO: Catch this exception
+    try:
+        shutil.copytree(str(source), str(destination))
+    except Exception as e:
+        raise CopyError(
+            f"Error: Failed to copy files from {source} "
+            + f"to {destination}.\n"
+            + f"{e}"
+        )
+
+
+# Translate an XML file using Unity's CLI
+def translate_file(unity_dir: Path, xml_path: Path):
+    logfile = Path(unity_dir, "Logs", f"cli_{xml_path.stem}.log")
+    try:
+        subprocess.run(
+            [
+                f"{UNITY_PATH}",
+                "-batchmode",
+                "-quit",
+                "-projectPath",
+                f"{unity_dir}",
+                "-logFile",
+                f"{logfile}",
+                "-executeMethod",
+                "CLI.Main",
+                "--xmlPath",
+                Path(*xml_path.parts[2:]),  # Path relative to unity_dir
+            ],
+            check=True,
+            capture_output=True,
```

Comment:
Do you need to capture output if you're not saving it to a variable?

Reply:
Or does this pass through the `Console.WriteLine` from C# to this program?

Reply:
Setting `capture_output=True` will set `stdin`, `stdout`, and `stderr` to `PIPE` under the hood, `text=True` converts the input to text